### PR TITLE
Optimized rss feed sql query

### PR DESCRIPTION
**Pull request by chrislombaard**

### CODE REVIEW

File: rssfeed/templatetags/rssfeed_tags.py [hunk 1]
```diff
@@ -7,5 +7,5 @@
 
 @register.inclusion_tag("rssfeed/inclusion_tags/render_rssfeed.html")
 def render_rssfeed(count=5):
-    entries = Entry.objects.all()[:count]
+    entries = Entry.objects.all()[:count].prefetch_related("feed")
```
Comment (jbeyers) [inline rssfeed/templatetags/rssfeed_tags.py]: prefetch_related must come before the list slice. Is there a test for this code path?

Comment (jbeyers) [reply rssfeed/templatetags/rssfeed_tags.py]: Should this not be select_related?

Comment (chrislombaard) [reply rssfeed/templatetags/rssfeed_tags.py]: You're absolutely right, select_related does only one query, whereas prefetch does 2.

Comment (chrislombaard) [reply rssfeed/templatetags/rssfeed_tags.py]: It doesn't matter where the select_related goes, django ORM is smart enough that it doesn't matter.

**Pull request merged**